.agents/SHIFT-LEFT.md

```diff
@@ -0,0 +1,287 @@
+# Shift-Left Validation Strategy
```

Comment:
@rjmurillo-bot This document shouldn't go in the root of `.agents`, find a better home for it